frontend/src/types/zielnorm.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * A `norms:zielnorm-reference`. The information about the Zielnorm and
+ * Geltungszeit of an amending command, see also: ADR-0015
+ */
+export type ZielnormReference = {
+  /** Typ of the reference (e.g. Änderungsvorschrift) */
+  typ: string
+
+  /** ID of the Geltungszeit */
+  geltungszeit: string
+
+  /** The eId of the associated amending command */
+  eId: string
+
+  /** Work ELI of the Norm that is targeted by the amending command */
+  zielnorm: string
```

Comment:
I think we have proper types for elis, but i'm not 100% sure they would work here